config/webhook/manifests.yaml

```diff
@@ -0,0 +1,70 @@
+
```

Comment:
not sure where this file belongs. It contains the CR definitions for the webhook configuration